src/Symfony/Component/Mailer/Transport/AbstractTransport.php

```diff
@@ -24,13 +24,15 @@
 use Symfony\Component\Mime\Address;
 use Symfony\Component\Mime\BodyRendererInterface;
 use Symfony\Component\Mime\RawMessage;
+use Symfony\Component\RateLimiter\RateLimiterFactoryInterface;
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
 abstract class AbstractTransport implements TransportInterface
 {
     private LoggerInterface $logger;
+    private ?RateLimiterFactoryInterface $rateLimiterFactory = null;
```

Comment:
New argument should be placed after to avoid BC breaks.

Reply:
Hm, how is this related to BC? This is just a private member variable - and not a constructor variable with constructor property promotion.

Reply:
Nevertheless changed in 8477ea1eb2